force-app/main/default/classes/ServiceService.cls

```diff
@@ -64,4 +67,42 @@ public with sharing class ServiceService {
         }
         return allServices;
     }
+
+    public Map<String, Object> getFieldSetsByName(String objectName) {
+        Integer indexToInsertServiceField;
+        Boolean fieldSetContainsServiceField = true;
+        Map<String, Object> serviceField = new Map<String, Object>();
+
+        Map<String, Object> fieldSets = fieldSetService.getFieldSetsByName(objectName);
+
+        for (String key : fieldSets.keySet()) {
+            indexToInsertServiceField = 0;
+            List<Map<String, Object>> fieldSetValues = (List<Map<String, Object>>) fieldSets.get(
+                key
+            );
+
+            for (Map<String, Object> field : fieldSetValues) {
+                String fieldApiName = (String) field.get('apiName');
+
+                if (fieldApiName == 'Contact__c') {
+                    indexToInsertServiceField = fieldSetValues.indexOf(field) + 1;
+                }
+                if (fieldApiName == 'ProgramEngagement__c') {
+                    indexToInsertServiceField = fieldSetValues.indexOf(field) + 1;
+                }
```

Comment:
Will these checks work for namespaced orgs?

Reply:
I think in Apex they do